epispot/models.py

```diff
@@ -49,9 +56,9 @@ def get_deriv(self, time, system):
         Return list of derivatives from each layer.
         Used by `integrate()` for evaluating model predictions.
 
-        :param time: Time to take derivative at
-        :param system: System of state values (S, I, R, etc.) --> e.g [973, 12, 15]
-        :return: List of derivatives in the order that layers were added
+        - time: Time to take derivative at
+        - system: System of state values (S, I, R, etc.) --> e.g [973, 12, 15]
+        - return: List of derivatives in the order that layers were added

```

Comment:
You should change all of the `:return:` directives to `- returns:` instead of `- return:`.

Reply:
Unnecessary